types/spectest/tests/ssvmsg/msg_id_belongs.go

```diff
@@ -7,14 +7,15 @@ import (
 
 // MsgIDBelongs tests msg id belonging to validator id
 func MsgIDBelongs() *SSVMessageTest {
-	return &SSVMessageTest{
-		Name: "belongs",
-		MessageIDs: []types.MessageID{
+	return NewSSVMessageTest(
+		"belongs",
+		"Test that message IDs with matching validator public key belong to the validator",
```

Comment:
I found this while resolving conflicts...
YOu are not using the constructor so you are overrding the Type field instead of the documentation...